_posts/2019-09-20-release-1.2.0.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: default
+title: 'Release v1.2.0'
+tags: ['v1.2.0', 'changelog']
+excerpt_separator: <!--more-->
+---
+As of 20/09 2019 the new version 1.2.0 is out
+<!--more-->
+
+## Menas Improvements
+- [#220](https://github.com/AbsaOSS/enceladus/issues/220) Editing old entities is disabled in the UI and the API returns a proper `400` resposne.
+- [#625](https://github.com/AbsaOSS/enceladus/issues/625) HDFS paths are not allowed to be empty or root `/`.
+- [#799](https://github.com/AbsaOSS/enceladus/issues/799) Non-string values in the schema's metadata no longer corrupts the schema entity. Only string values are allowed in the schema's metadata.  
+- [#849](https://github.com/AbsaOSS/enceladus/issues/849) Improved schema upload API so it now returns a more structured and appropriate `400` response if an error has occurred. 
+- [#491](https://github.com/AbsaOSS/enceladus/issues/491) Fixed `TypeError` when session expires.
+- [#863](https://github.com/AbsaOSS/enceladus/issues/863) Fixed front-end logging issue. 
+
+## Standardization Improvements
+- [#909](https://github.com/AbsaOSS/enceladus/issues/909) Allow standardization to process empty datasets. Non-empty datasets which are read incorrectly (e.g., `xml` with a wrong `row tag`) will still fail the run if their source/raw checkpoint says they are supposed to have more than 0 records.
+
+## Conformance Improvements
+- [#427](https://github.com/AbsaOSS/enceladus/issues/427) Drop rule checks for the presence of the column before dropping it and logs a warning if not found.
+- [#822](https://github.com/AbsaOSS/enceladus/issues/822) Nullable fields which fail during `conformance` will have `null` values rather then be assigned a global default.
```

Comment:
```suggestion
- [#822](https://github.com/AbsaOSS/enceladus/issues/822) Nullable fields which fail during `conformance` will have `null` values rather than be assigned a global default.
```